pyof/v0x01/common/action.py

```diff
@@ -116,6 +116,7 @@ class ActionOutput(ActionHeader):
     max_length = UBInt16()
 
     _allowed_types = ActionType.OFPAT_OUTPUT,
+    _removed_attributes = ['pad']
```

Comment:
Please, lets use _removed_attributes only when implementing a new version of the spec.

Remove pad from `action_header`, since that the specs is messy.